examples/kubernetes/dynamic_provisioning/specs/storageclass.yaml

```diff
@@ -7,5 +7,8 @@ parameters:
   subnetId: subnet-0930cbd463b9688c5
   securityGroupIds: sg-03a4b6bd8afb19051,sg-05a37bfe01467059a
   deploymentType: PERSISTENT_1
+  automaticBackupRetentionDays: "1"
```

Comment:
Since these fields are optional, does FSX provides default value? I mean if we need to give all configurations in SC

Reply:
yes, default is 7 days